src/tic_tac_toe/play_game.clj

```diff
@@ -1,5 +1,7 @@
 (ns tic-tac-toe.play-game) 
 (use '[tic-tac-toe.console])
+(use '[tic-tac-toe.board])
 
 (defn -main []
-  (print-empty-board))
+ (print-welcome-message) (print-empty-board) (ask-for-move) (print-board (make-move ["-" "-" "-" "-" "-" "-" "-" "-" "-"] (get-move) "X")))
```

Comment:
why are you instantiating an empty board here? you should be doing that with the `board/empty-board` function